tcf_website/templates/course/course_professor.html

```diff
@@ -183,15 +183,17 @@ <h2 class="mr-md-3">{{ course.title }}</h2>
                         </div>
                     </div>
                 </div>
-
                 <div class="grades-container">
                     <div class="chart-container d-flex align-items-center justify-content-center">
-                        <canvas width="100%" height="100%" id="myChart"></canvas>
+                        <div id="canvas-parent"><canvas id="myChart"></canvas></div>
                     </div>
-                    <div class="absolute-center">
+                    <div class="absolute-center" id="toggleinformation">
                         <p class="gpa-text"> No grades found <i class="far fa-frown-open fa-fw"></i></p>
                         <p class="students-text"> &mdash; Students</p>
                     </div>
+                    <div class="lower-button">
+                        <button type="button" id="toggle-btn" class="btn btn-outline-primary pieToBar" value="bar">Bar</button>
+                    </div>
```

Comment:
This might be a bit of a nitpick, but it may be good to change `toggleinformation` to something like `chart-label` and `lower-button` to `chart-button` or similar